lib/Doctrine/DBAL/Schema/Column.php

```diff
@@ -21,13 +20,13 @@ class Column extends AbstractAsset
     protected $_type;
 
     /** @var int|null */
-    protected $_length = null;
+    protected $_length;
 
     /** @var int */
-    protected $_precision = 10;
+    protected $_precision;
 
     /** @var int */
-    protected $_scale = 0;
```

Comment:
Same here: default value to be preserved IMO

Reply:
Reverted from this patch.